app/utils/versionChecker.js

```diff
@@ -1,18 +1,15 @@
 class VersionChecker {
-  latest (callback) {
```

Comment:
Here was a mix of callback approach and promise. But it's not a big issue, the main problem was that method handled the exception on its own. So users of the method couldn't catch the actual exception and handle it properly. Moreover, they couldn't continue a chain of promises. Also, we could use async await here but it involves an update of the node.